client/assets/scss/main.scss

```diff
@@ -12,3 +16,180 @@
   -moz-tab-size: 4;
   tab-size: 4;
 }
+
+h1, h2, h3, h4, h5, h6, p {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  color: $navy-blue;
+  background-color: snow;
+}
+
+.st0 {
+  fill:#22355C;
+}
+
+.st1 {
+  fill:#FFAF00;
+}
+
+#top-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 5%;
+  h3 {
+    font-family: 'Poppins', sans-serif;
+    color: $navy-blue;
+    font-weight: bold;
+  }
+  h6 {
+    color: gray;
+  }
+}
+
```

Comment:
Add spaces between the nested codes blocks in this file.  For example an empty line between 45&46 and 50&51, so we end up with 

```scss
#top-header {
  display: flex;
  justify-content: space-between;
  padding: 10px 5%;
  
  h3 {
    font-family: 'Poppins', sans-serif;
    color: $navy-blue;
    font-weight: bold;
  }
  
  h6 {
    color: gray;
  }
}
```

Please apply these principles to all the code blocks below too